docs/docs/fundamentals/installation.md

```diff
@@ -4,98 +4,129 @@ title: Installation
-`gestureHandlerRootHOC`.
+#### Android
 
-For example:
+Android doesn't require any more steps, but keep in mind that if you want to use gestures in Modals you need to wrap Modal's content with `GestureHandlerRootView`:
 
-```js
-import { GestureHandlerRootView } from 'react-native-gesture-handler';
+```jsx
+import { Modal } from 'react-native';
+import { GestureHandlerRootView } from 'react-native-gesture-handler`;
 
-export default function App() {
+export function CustomModal({ children, ...rest }) {
   return (
-    <GestureHandlerRootView style={{ flex: 1 }}>
-      {/* content */}
-    </GestureHandlerRootView>
+    <Modal {...rest}>
+      <GestureHandlerRootView>
+        {children}
+      </GestureHandlerRootView>
+    </Modal>
   );
 }
 ```
 
-:::info
-If you use props such as `shouldCancelWhenOutside`, `simultaneousHandlers`, `waitFor` etc. with gesture handlers, the handlers need to be mounted under a single `GestureHandlerRootView`. So it's important to keep the `GestureHandlerRootView` as close to the actual root view as possible.
 
-Note that `GestureHandlerRootView` acts like a normal `View`. Also, gestures will only be recognized within the `GestureHandlerRootView` area. So if you want it to fill the screen, you will need to pass `{ flex: 1 }` like you'll need to do with a normal `View`. By default, it'll take the size of the content nested inside.
-:::
+##### Kotlin
 
-:::tip
-If you're using gesture handler in your component library, you may want to wrap your library's code in the GestureHandlerRootView component. This will avoid extra configuration for the user.
-:::
+Gesture Handler on Android is implemented in Kotlin. If you need to set a specific Kotlin version to be used by the library, set the `kotlinVersion` ext property in `android/build.gradle` file and RNGH will use that version:
 
-### Linking
+```groovy
+buildscript {
+    ext {
+        kotlinVersion = "1.6.21"
+    }
+}
+```
+
+#### iOS
 
-> **Important**: You only need to do this step if you're using React Native 0.59 or lower. Since v0.60, linking happens automatically.
+While developing for iOS, make sure to install [pods](https://cocoapods.org/) first before running the app:
 
 ```bash
-react-native link react-native-gesture-handler
+cd ios && pod install && cd ..
 ```
 
-## Fabric
-
-Starting with version 2.3.0, Gesture Handler now supports [Fabric](https://reactnative.dev/docs/fabric-renderer)!. To use Gesture Handler in your Fabric application you have to:
+#### Web
 
-#### on iOS:
+There is no additional configuration required for the web, however, since the Gesture Handler 2.10.0 the new web implementation is enabled by default. It is recommended to check if the gestures in your app are working as expected since their behavior should now resemble the native platforms. If you don't want to use the new implementation, you can still revert back to the legacy one by enabling it at the beginning of your `index.js` file:
```

Comment:
```suggestion
There is no additional configuration required for the web, however, since the Gesture Handler 2.10.0 the new web implementation is enabled by default. We recommend you to check if the gestures in your app are working as expected since their behavior should now resemble the native platforms. If you don't want to use the new implementation, you can still revert back to the legacy one by enabling it at the beginning of your `index.js` file:
```
seems a bit more friendly